cpp/tests/wrappers/bools_test.cpp

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/wrappers/bool.hpp>
+
+#include <tests/utilities/cudf_gtest.hpp>
+
+#include <random>
+
+struct Bool8Test : public ::testing::Test {
```

Comment:
You can use the `UniformRandomGenerator` base class to generate random numbers for you instead of writing your own logic. 
```suggestion
struct Bool8Test : public cudf::test::BaseFixture, cudf::test::UniformRandomGenerator {
```

Reply:
Sweet thanks, I didn't know that existed. Made a small change to allow generating bools in https://github.com/rapidsai/cudf/pull/3087/commits/a00ce5a92c81b1a6f1b06b463cc0b5ca6de4e28a, and https://github.com/rapidsai/cudf/pull/3087/commits/a6673ecacc81a14743209559de2f83db7cdf2c48 updates the bool8 tests to use it.